### DISCUSSION TITLE
ProFormDigit 传入 precision 后, readonly 模式下保留位数是否需要保持一致?

### DISCUSSION THREAD

**Discussion by conorzhong** [ant-design/pro-components#5359] [Ideas]
https://codesandbox.io/s/confident-frost-z38ict?file=/App.tsx
<img width="229" alt="image" src="https://user-images.githubusercontent.com/39456642/172789693-74da334f-e5f4-4298-8e3a-7db28fb9e94c.png">

**Comment by chenshuai2144** [comment]
应该保持一致的
